packages/perseus-editor/src/article-editor.tsx

```diff
@@ -64,11 +70,24 @@ export default class ArticleEditor extends React.Component<Props, State> {
 
     constructor(props: Props) {
         super(props);
+
+        // Check if the json needs to be converted
+        const conversionWarningRequired = conversionRequired(
+            props.json as PerseusRenderer,
+        );
```

Comment:
This is probably not be safe. I'm pretty sure there are places in webapp where it passes in `ReadonlyArray<PerseusRenderer>` (essentially the `PeprseusArticle` type). I think it can be fixed fairly easily though with: 

```suggestion
        const conversionWarningRequired =
            props.json instanceof Array
                ? props.json
                      .map(conversionRequired)
                      .reduce((p, v) => p && v, false)
                : conversionRequired(props.json);
```

Reply:
Oh great catch, thank you! I forgot to check for the sections as part of the conversionRequired check